parsers/NA.py

```diff
@@ -52,6 +50,31 @@ def get_text_from_image(link, expected_size, new_size, logger, session=None):
     return text
 
 
+def timestamp_checker(session=None, logger=None):
+
+    scada_info = get_text_from_image(session=session, link=timestamp_link, \
+                                    expected_size=(600,20), new_size=(1200,40), \
+                                    logger=logger)
+
+    timestamp = scada_info.split(':', 1)[1]
+
+    try:
+        scada_time = arrow.get(timestamp, ' DD/MM/YYYY HH:mm:ss')
+    except arrow.parser.ParserError as e:
+        logger.warning('Namibia scada timestamp cannot be read, got {}.'.format(timestamp))
```

Comment:
you should probably fail here, no? you won't be able to use `scada_time` later anyway

Reply:
If it fails then we lose the datapoint entirely. The OCR of this timestamp is not super reliable (small, stretched, poor font) so we'd be dropping valid data quite often.

Reply:
:+1: